homeassistant/components/letpot/strings.json

```diff
@@ -49,6 +49,29 @@
         "name": "Refill error"
       }
     },
+    "select": {
+      "display_temperature_unit": {
+        "name": "Display temperature unit",
+        "state": {
+          "celsius": "Celsius",
+          "fahrenheit": "Fahrenheit"
+        }
+      },
+      "light_brightness": {
+        "name": "Light brightness",
+        "state": {
+          "low": "[%key:common::state::low%]",
+          "high": "[%key:common::state::high%]"
+        }
+      },
+      "light_mode": {
+        "name": "Light mode",
+        "state": {
+          "flower": "Fruits / flowers",
+          "vegetable": "Veggie / herbs"
```

Comment:
How does the integration call these?

Reply:
I don't understand your question.

The enums are based on bit returned by the device (0/1), comverted by the library to make it more descriptive.

These strings match the names used by the manufacturer in the app.

Reply:
> These strings match the names used by the manufacturer in the app.

This is the answer :)

Reply:
Slightly updated strings to better [match style guidelines](https://learn.microsoft.com/en-us/style-guide/punctuation/slashes#:~:text=Capitalize%20the%20word%20after%20the%20slash%20if%20the%20word%20before%20the%20slash%20is%20capitalized.%20For%20example%2C%20if%20country/region%20is%20used%20as%20a%20label%20in%20a%20form%2C%20capitalize%20it%20as%20Country/Region.).

For future reference, manufacturer app option for this entity:
<img width="1080" height="1655" alt="image" src="https://github.com/user-attachments/assets/95ef58a8-382c-4f1b-8eab-a06f1091eafc" />